docs/design-notes/rust.md

```diff
@@ -0,0 +1,38 @@
+# Rust and C++
+
+Rust introduces many useful features that are missing in C++. This document
+shows how to use them in Scylla.
+
+## Using Rust in Scylla
+
+To create a Rust module `new_pkg` and use it in a C++ source file:
```

Comment:
`s/module/package/`

"Module" and "package" mean different things in the context of Rust. The correct word here is "package".

Reply:
Ok